realm/realm-library/src/androidTest/java/io/realm/LinkingObjectsManagedTests.java

```diff
@@ -630,14 +633,15 @@ public void migration_backlinkedSourceFieldDoesntExist() {
      * for `BacklinksWrongTypeTarget` should notice, though, that its `parents` field points to an object
      * of the wrong type, `Integer`, instead of `BacklinksWrongTypeSource`.
      */
+    @Ignore("Need to rebuild the test library")
```

Comment:
Some scenarios might be too much trouble to be worth unit testing. This is such a far fetched corner case anyway, and the user would it immediately when using the app. So not having these tests are probably fine 👍